src/osd/OSDMap.h

```diff
@@ -275,12 +275,10 @@ class OSDMap {
   }
 
   // no copying
-  /* oh, how i long for c++11...
 private:
   OSDMap(const OSDMap& other) = default;
```

Comment:
if we want to disable the default copy ctor and assign operator, we might want to put 
```c++
OSDMap(const OSDMap& other) = delete;
OSDMap& operator=(const OSDMap& other) = delete;
```
instead, and probably remove the `private:` before them also.